lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
OK. Let me change it to prefetch.

19 SQL queries are due to rendering of team dashboard which includes getting topics against course and then adding teams count to those topics. This is a prior implementation.